py-polars/docs/source/reference/lazyframe/miscellaneous.rst

```diff
@@ -10,7 +10,6 @@ Miscellaneous
     LazyFrame.collect
     LazyFrame.collect_async
     LazyFrame.collect_schema
-    LazyFrame.fetch
```

Comment:
Normally I like to keep deprecated functions around in the docs, but in this case I agree removing it is better.